pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -1052,6 +1095,11 @@ func (l *LibvirtDomainManager) SyncVMI(vmi *v1.VirtualMachineInstance, useEmulat
 				return nil, err
 			}
 			logger.Info("Domain defined.")
+			err = l.generateCloudInitISO(vmi, &dom)
```

Comment:
minor "gotcha" here. If somehow generateCloudInitIso() fails here. The next time this SyncVMI function is called, this function won't be called again because the domain will have already been defined.

Reply:
@davidvossel good point. we can move it to line 1128, just before the create(). What do you think?

Reply:
yeah, it seems like that would work

Reply:
Done